### DISCUSSION TITLE
Add booking from backend event view RSVP

### DISCUSSION THREAD

**Discussion by carlbannister** [liedekef/events-made-easy#282] [Q&A] [answered]
Hi, I am logged into the back end looking at an event and using the RSVP shortcut to add a booking. Is there a way i can pull the data from the people database instead of having to type it all in manually?

**Comment by liedekef** [comment]
The people info is auto-complete (just not the custom fields I think), so start typing the last name and then select the one you want.

**Reply by carlbannister** [reply]
When i try this it looks like its searching then doesn't return anything. Tried last name and first name

**Reply by liedekef** [reply]
I tried it locally and on a customer install: works fine. What does your browser console say? Maybe something is blocking the ajax call (javascript from your browser to the backend)?

**Reply by carlbannister** [reply]
![image](https://github.com/liedekef/events-made-easy/assets/143013419/44f66f5b-1875-4504-9511-f652005720b8)

**Reply by liedekef** [reply]
Seems you have javascript errors in your backend, you need to get those fixed. The red error is not complete, but it seems related to a plugin called "hostinger"

**Reply by carlbannister** [reply]
Ive cleared the plugins that were creating errors but its still not working and i cannot see any issues in the console. Any other ideas?

![image](https://github.com/liedekef/events-made-easy/assets/143013419/7a3110ab-5e5d-409b-9a7f-a9cece2e3691)

**Reply by liedekef** [reply]
then, in your console, go in the "network" tab. After that, start typing a last name and see what is being sent/received.
If that's too complicated, I can take a look for you on your end too.

**Reply by liedekef** [reply]
No further reply, do considering solved.

**Reply by carlbannister** [reply]
Hi Franky, sorry for the late reply, i have been away all weekend.
I have tried your instruction from the network tab and get this. I cannot see anything blocking.
Happy to give you a login if that is easier.
Many thanks for the support
![image](https://github.com/liedekef/events-made-easy/assets/143013419/5c83621d-5160-4280-b7ce-b66f7f5d4308)

**Reply by liedekef** [reply]
Feel free to send me an account via mail: franky@e-dynamics.be
Also: install the plugin "Health Check & Troubleshooting" , with which you can switch to a default theme and deactivate all plugins just for the current session (so no impact on other users) and that allows you then to activate one plugin by one (or switch theme) until you can see which plugin causes problems.

**Comment by f14m07** [comment]
Hello Franky,
I'm having the same problem on my production site, but it's working on the local site.
When I enter the first few letters of the name, the little search logo appears but no name is suggested. On the booking form, on the other hand, the search works very well.
So I followed the instructions on the network tab and I got a server error 500 on the following request:
```Request URL:
https://vogue-et-reve.fr/wp-admin/admin.php?page=eme-registration-seats&eme_admin_action=newBooking&event_id=619&eme_admin_nonce=e7f6b2c4a3&eme_notice_ignore=donate

The server response is :
{
    "code": "internal_server_error",
    "message": "<p>A critical error has occurred on this site. Please consult your site's admin email inbox for more information."</p><p><a href="https:\/en.wordpress.org/support\/article/debugging-in-wordpress">Learn more about WordPress bugging.</a><\/p>",
    "data": {
        "status": 500
    },
    "additional_errors": []
}
Do you need other information from the inspector?
Any idea?

I haven't received any e-mail from WP

**Reply by liedekef** [reply, marked as answer]
The issue for the original problem here was that the "autocomplete sources" setting in the general EME settings was set to "none", so in fact it didn't do anything. Changing the setting to e.g. "EME people" caused it to work as expected. Your internal error needs to be checked in the php logfiles.

**Comment by f14m07** [comment]
My setting "autocomplete sources" was good. Here is the php error:
```
[05-Sep-2023 13:31:04 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function get_current_screen() in /home/taeq4768/vogue-et-reve.fr/wp-content/plugins/code-snippets/php/snippet-ops.php(575) : eval()'d code:45
Stack trace:
#0 /home/taeq4768/vogue-et-reve.fr/wp-includes/class-wp-hook.php(310): smartwp_sort_last_login_column()
#1 /home/taeq4768/vogue-et-reve.fr/wp-includes/class-wp-hook.php(334): WP_Hook->apply_filters()
#2 /home/taeq4768/vogue-et-reve.fr/wp-includes/plugin.php(565): WP_Hook->do_action()
#3 /home/taeq4768/vogue-et-reve.fr/wp-includes/class-wp-user-query.php(281): do_action_ref_array()
#4 /home/taeq4768/vogue-et-reve.fr/wp-includes/class-wp-user-query.php(79): WP_User_Query->prepare_query()
#5 /home/taeq4768/vogue-et-reve.fr/wp-content/plugins/events-made-easy/eme-people.php(4182): WP_User_Query->__construct()
#6 /home/taeq4768/vogue-et-reve.fr/wp-content/plugins/events-made-easy/eme-people.php(4682): eme_get_wp_users()
#7 /home/taeq4768/vogue-et-reve.fr/wp-content/plugins/events-made-easy/eme-actions.php(118): eme_people_autocomplete_ajax()
#8 /home/taeq4768/vogue-et-reve.fr/wp-includes/class-wp-hook.php(310): eme_actions_early_init()
#9 /home/taeq4768/vogue-et-reve.fr/wp-includes/class-wp-hook.php(334): WP_Hook->apply_filters()
#10 /home/taeq4768/vogue-et-reve.fr/wp-includes/plugin.php(517): WP_Hook->do_action()
#11 /home/taeq4768/vogue-et-reve.fr/wp-settings.php(564): do_action()
#12 /home/taeq4768/vogue-et-reve.fr/wp-config.php(93): require_once('/home/taeq4768/...')
#13 /home/taeq4768/vogue-et-reve.fr/wp-load.php(50): require_once('/home/taeq4768/...')
#14 /home/taeq4768/vogue-et-reve.fr/wp-admin/admin.php(34): require_once('/home/taeq4768/...')
#15 {main}
  thrown in /home/taeq4768/vogue-et-reve.fr/wp-content/plugins/code-snippets/php/snippet-ops.php(575) : eval()'d code on line 45

I have no snippet for a RSVP action. And the 'completion name' works well on the RSVP form

**Reply by liedekef** [reply]
Not EME related, see the first line of your output :
 PHP Fatal error:  Uncaught Error: Call to undefined function get_current_screen() in /home/taeq4768/vogue-et-reve.fr/wp-content/plugins/code-snippets/php/snippet-ops.php(575) : eval()'d code:45

**Comment by f14m07** [comment]
Thank you for helping me identify the culprit. In fact it's a snippet that I copied and pasted from the internet to memorise the dates of members' logins to WP. There is no reference to a table or an EME function. It's all linked to the way WP works. By deactivating this snippet everything works normally. It's odd, though, that this should have an impact on EME. Thanks again for your efficient support

**Reply by liedekef** [reply]
Snippets can run in the frontend, backend or always. So depending on the code in it and where it needs to run, it can interfere with EME or any other plugin.
